src/modules/account/RadixAccount.ts

```diff
@@ -47,14 +57,41 @@ export default class RadixAccount {
     public messagingSystem: RadixMessagingAccountSystem
     public tokenDefinitionSystem: RadixTokenDefinitionAccountSystem
 
-    private syncedSubject = new BehaviorSubject(false)
     private processingAtomCounter = new BehaviorSubject(0)
 
     private atomObservable: Observable<RadixAtomObservation>
 
+    private subs = new Subscription()
+
+    private sleepAmount = sleepAmountIncrease
+
+
+    public unsubscribeSubscribers() {
```

Comment:
Nice, I'm surprised we weren't handling subscriptions before 🙀

Reply:
yeah we had like 100 memory leaks.... :P